tests/foreman/api/test_activationkey.py

```diff
@@ -173,8 +173,12 @@ def test_positive_update_limited_host(max_host):
     :parametrized: yes
     """
     # unlimited_hosts defaults to True.
-    act_key = entities.ActivationKey().create()
-    want = {'max_hosts': max_host, 'unlimited_hosts': False}
+    act_key = target_sat.api.ActivationKey().create()
+    want = {
+        'max_hosts': max_host,
+        'unlimited_hosts': False,
+        'organization_id': act_key.organization_id,
```

Comment:
Why does AK update requires organization id everytime? is there some dependancy on it?

Reply:
its a required parameter for that call

``` Received HTTP 404 response: {"displayMessage":"One of parameters [ organization_id ] required but not specified."```